use_cases/examples/flashing_detection/data/schema/example.tf

```diff
@@ -0,0 +1,178 @@
+provider "google" {
```

Comment:
Could we add a copyright here? I appreciate it is a bit over the top, but it would then be consistent with for example this:
https://github.com/gts-trading-analytics/reg-reporting-blueprint/blob/main/common_components/orchestration/infrastructure/main.tf